config/initializers/sample_manifest_excel.rb

```diff
@@ -9,3 +9,11 @@
     end
   end
 end
+
+Rails.application.config.tube_manifest_barcode_config = {
+  barcode_type_labels: {
+    '1d' => '1D Barcode (with machine readable barcode encoded)',
+    '2d' => '2D Barcode (with human readable barcode encoded)'
+  },
+  two_dimensional_label_template: 'traction_tube_label_template'
+}
```

Comment:
I’m not 100% sure this is the best place for this config, but it probably doesn’t really matter